src/plugins/Router.ts

```diff
@@ -73,6 +74,26 @@ const routes: Array<RouteRecordRaw> = [
             app: AppType.Quantum,
         },
     },
+    {
+        path: '/quantum/options/:path/:shareId?',
```

Comment:
this needs to be implemented in normal quantum flow, should not be a diffrent path/route